include/wlr/types/wlr_keyboard.h

```diff
@@ -63,7 +63,7 @@ enum wlr_key_state {
 };
 
 struct wlr_event_keyboard_key {
-	uint32_t time_sec;
+	uint32_t time_msec;
```

Comment:
Does it have to be a uint32_t ?
2^32 msec is "only" 49-50 days, I could totally see folks (e.g. myself) keep a computer hibernated for that long, and wakeup might feel weird... Well I guess as far as wlroots is concerned it'll just be far over whatever threshold we're considering anyway, but if time_msec replaces both time_sec and time_usec, and is computed from time_usec, I'd keep it as uint64_t
(heck, I'd do everything as usec if it's not too much of a pain, but the design is around msecs so sure might as well go for it now)